src/status_im/ethereum/core.cljs

```diff
@@ -138,3 +138,13 @@
                          nil)]
     (when normalized-key
       (subs (sha3 normalized-key) 26))))
+
+(def ^:const bytes32-length 66) ; length of '0x' + 64 hex values. (a 32bytes value has 64 nibbles)
```

Comment:
but what if text message has a 66 length?

Reply:
It will be shown as a hex string. I don't think it's possible to identify if a hex value should be shown as a string or as a hexadecimal string (EIP712 fixes that with typed data). This assumes that it's more common that a 32 bytes hex string will be represented as such and not as a string value. 

For comparison with Metamask, that implements this same behavior, I tried executing this instruction - Signing a string with 32 characters - and it's represented as a hex value:

```
web3.eth.personal.sign("ABCDEFGHABCDEFGHABCDEFGHABCDEFGH", web3.eth.defaultAccount);
```

![Screenshot from 2020-04-23 10-37-36](https://user-images.githubusercontent.com/1106587/80112251-e5594e80-854e-11ea-8ab8-ba55a3d6c73a.png)

However, just removing/adding one character will make the string be displayed as text

![Screenshot from 2020-04-23 10-37-17](https://user-images.githubusercontent.com/1106587/80112257-e68a7b80-854e-11ea-9c95-d55c96ac6167.png)
![Screenshot from 2020-04-23 10-39-02](https://user-images.githubusercontent.com/1106587/80112365-0752d100-854f-11ea-8801-abef4cc4356c.png)

Reply:
that's weird, mhm dunno what to do